analyzers/maven/maven_test.go

```diff
@@ -0,0 +1,143 @@
+package maven_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/fossas/fossa-cli/analyzers/maven"
+	"github.com/fossas/fossa-cli/module"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+func TestMavenPOMFile(t *testing.T) {
+	buildTarget := ""
+	m := module.Module{
+		Name:        "test",
+		Type:        pkg.Maven,
+		BuildTarget: buildTarget,
+		Dir:         "test-data",
+	}
+
+	mavenModule := m
+
+	analyzer, err := maven.New(mavenModule)
+	assert.NoError(t, err)
+
+	analyzed, err := analyzer.Analyze()
+	assert.NoError(t, err)
+
+	directDependencies := analyzed.Direct
+
+	compileTimeDependencyfound := false
+	runtTimeDependencyFound := false
+	testDependencyFound := false
+	for dependencyIndex := 0; dependencyIndex < len(directDependencies); dependencyIndex++ {
```

Comment:
You can use `for _, dep := range directDependencies {}` as a for-each loop in Go.

Reply:
Did it 👍